_data/concepts.yml

```diff
@@ -47,7 +47,6 @@ toc:
     - docs/concepts/workloads/controllers/replicationcontroller.md
     - docs/concepts/workloads/controllers/deployment.md
     - docs/concepts/workloads/controllers/statefulset.md
-    - docs/concepts/workloads/controllers/petset.md
```

Comment:
Not sure a change to `_data` is needed or not.

Reply:
should update when you build the documentations

Reply:
thanks for verifying that a change to `_data` is needed. :)